api/lds.proto

```diff
@@ -4,6 +4,8 @@
 
 syntax = "proto3";
 
+package envoy.api.v1beta1;
```

Comment:
Why are we bothering with beta1 at all? Can we just start with v2 and iterate? It's basically what we will implement and ship and then it becomes v2.